File: problem/Min_Distance_Target/getMinDistance.c
#include <stdlib.h>
#include <math.h>

int getMinDistance(int *nums, int numsSize, int target, int start)
{
    int result = numsSize;
    for (int i = 0; i < numsSize; i++)
    {
        if (nums[i] == target)
        {
            int diff = abs(i - start);
            if (diff < result)
            {
                result = diff;
            }
        }
    }
    return result;
}

File: problem/Min_Distance_Target/getMinDistance.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @param {number} start
 * @return {number}
 */
var getMinDistance = function(nums, target, start) {
    let result = nums.length;
    for (let i = 0; i < nums.length; i++) {
        if (nums[i] === target) {
            result = Math.min(result, Math.abs(i - start));
        }
    }
    return result;
};